src/type_manipulation.cpp

```diff
@@ -0,0 +1,54 @@
+#include "type_manipulation.hpp"
+
+namespace RobotArm {
+char *TypeManipulation::stradd(char *dest, const char *src) {
+    size_t i = 0, j = 0;
+
+    for (i = 0; dest[i] != '\0'; i++) {
+    }
+
+    for (j = 0; src[j] != '\0'; j++) {
+        dest[i + j] = src[j];
+    }
+
+    dest[i + j] = '\0';
+    return dest;
+}
+
+char *TypeManipulation::strcopy(char *dest, const char *src) {
```

Comment:
Why return a char? It's a pointer that is passed as a parameter, so the recipient already knows it.

Reply:
Again Fair point.